backend/index.ts

```diff
@@ -0,0 +1,117 @@
+import Fastify from 'fastify';
+import cors from '@fastify/cors';
+
+import fastifyStatic from '@fastify/static';
+import swagger from '@fastify/swagger';
+import swaggerUi from '@fastify/swagger-ui';
+import path from 'path';
+import routes from './routes';
+import { initializeDb } from './utils/db';
+import config from './config';
+import mercurius from 'mercurius';
+import { typeDefs } from './graphql/schema';
+import { resolvers } from './graphql/resolvers';
+import MercuriusGQLUpload from 'mercurius-upload';
+import fastifyMultipart from '@fastify/multipart';
+
+async function start() {
+   try {
+      // Log configuration on startup
+      console.log(`Starting server in ${config.server.env} mode`);
+
+      // Initialize database
```

Comment:
Redundant comment, we can understand what's going on from the name of the function that is being called.